lib/rules/indentation/html.test.js

```diff
@@ -137,24 +149,27 @@ a {
 			],
 		},
 		{
-			code: `
-\t<style>
-    a {
-        display:block;
-    }
-    b {
-      display:block;
-    }
-\t</style>`,
-			fixed: `
-\t<style>
-\ta {
-\t\tdisplay:block;
-\t}
-\tb {
-\t\tdisplay:block;
-\t}
-\t</style>`,
+			code: stripIndent(`
+					<style>
+				    a {
+					display:block;
+				    }
+				    b {
+				      display:block;
+				    }
+					</style>
+			`),
+
+			fixed: stripIndent(`
```

Comment:
I also think that specifically this test would be better to keep as it was - `stripIndent` could theoretically affect the interaction between the mix of tabs and spaces